README.md

```diff
@@ -315,6 +315,18 @@ setVerbose(True)
 # turn off verbose mode
 setVerbose(False)
 ```
+### Bypassing merge task 
+
+Skipping the merge task is useful when the tasks results don't need to be merge into a list. To bypass the merge task, you can pass the *enableMerge* flag to the foreach object:
+
+```R
+# Enable merge task
+foreach(i = 1:3, .options.azure = list(enableMerge = TRUE))
+
+# Disable merge task
+foreach(i = 1:3, .options.azure = list(enableMerge = FALSE))
+```
+Note: User defined functions for the merge task is on our list of features that we are planning on doing.

```

Comment:
So, when the UDF is available, we'll extend the merge task to be cbind, rbind, c, etc?

What is the default merge task today? (add to the docs). Also, how does one change the merge task from c to cbind or rbind (Add to the docs)